README

```diff
@@ -334,6 +334,16 @@ in test failure descriptions. Very useful in combination with MonkeyPatch.
   ...     pass
   >>> fixture.cleanUp()
 
+Deprecations
+++++++++++++
+
+Prevent code under test from calling deprecated functions.
+
```

Comment:
You'll need to import warnings here.

You can make this a little cleaner by writing the test using with (we no longer support 2.5)
```
>>> with fixture:
...    warnings.warn('stop using me', DeprecationWarning)
Exception...
```